scipy/spatial/tests/test_spherical_voronoi.py

```diff
@@ -170,3 +171,24 @@ def test_single_hemisphere_handling(self):
             dots = np.einsum('ij,ij->i', sv.vertices, triangles[:, 0])
             circumradii = np.arccos(np.clip(dots, -1, 1))
             assert np.max(circumradii) > np.pi / 2
+
+    @pytest.mark.parametrize("radius,center",
+                             itertools.product([0.5, 1, 2],
+                                               [(0, 0, 0), (1, 2, 3)]))
+    def test_areas(self, radius, center):
+        for points in [self.points, self.hemisphere_points,
+                       self.hemisphere_points2]:
+            sv = SphericalVoronoi(radius * points + center,
+                                  radius=radius,
+                                  center=center)
+            areas = sv.calculate_areas()
+            assert_almost_equal(areas.sum(), 4 * np.pi * radius**2)
+
+    @pytest.mark.parametrize("radius,center",
+                             itertools.product([0.5, 1, 2],
+                                               [(0, 0, 0), (1, 2, 3)]))
+    def test_equal_areas(self, radius, center):
+        points = radius * np.concatenate((-np.eye(3), np.eye(3))) + center
```

Comment:
maybe add a comment here -- looks like you're checking the surface area footprint [projected from a cube](http://mathworld.wolfram.com/SolidAngle.html). Can you add a comment of this nature for clarity? And maybe a more specific test name to go with it?

Reply:
I have extended this test and renamed it `test_area_reconstitution_polyhedron`.  It now uses all the regular polyhedra as a sanity check.